certbot/main.py

```diff
@@ -235,6 +215,13 @@ def _find_lineage_for_domains(config, domains):
     elif subset_names_cert is not None:
         return _handle_subset_cert_request(config, domains, subset_names_cert)
 
+def _find_cert(config, domains, certname):
+    """Returns a tuple of (should_get_cert, lineage or None)"""
```

Comment:
Ok so I'll definitely mention the type (boolean). Beyond that - "Finds an existing certificate object given domains and/or a certificate name. `should_get_cert` indicates if this function should be followed by a call to fetch a certificate from the server."